monitor-manage-device-with-iotcentral/Parameters.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using CommandLine;
+
+namespace Learn.CoffeeMaker
+{
+    /// <summary>
+    /// Command line parameters for the SimulatedDevice sample
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'i',
+            "DpsIdScope",
+            HelpText = "(Required if DeviceSecurityType is 'dps'). \nThe DPS ID Scope to use during device provisioning." +
```

Comment:
For IoT Central - dps is the only security type. You could reword this here and below. You don't even need to mention DPS. In the IoT Central UI, this value is simply "ID scope"

Reply:
Changed